src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -482,6 +483,7 @@ private function addSessionSection(ArrayNodeDefinition $rootNode)
                         ->scalarNode('cookie_domain')->end()
                         ->booleanNode('cookie_secure')->end()
                         ->booleanNode('cookie_httponly')->defaultTrue()->end()
+                        ->enumNode('cookie_samesite')->values(array(null, Cookie::SAMESITE_LAX, Cookie::SAMESITE_STRICT))->defaultNull()->end()
```

Comment:
what about setting it to lax by default? this would improve the security of all Symfony apps by default
alternatively, we should make it the default in the flex recipe

Reply:
That would be a BC as it could break some apps that post across domains by design.

Setting it as default in the flex recipe sounds good though!

Reply:
Does this use case really exist? It would be great to improve the security of everyone by default if not. Worth thinking twice here.

Reply:
Maybe just put it in the flex recipe for now and default it to Lax in the framework for Symfony 5?

Reply:
If we really think this is a BC break, we cannot change it in 5 without first issuing a notice in 4...
Can you add a strong recommendation in the upgrading file also btw?

Reply:
@rpkamp, could you please add a note in the upgrading file to suggest enabling the lax mode?

Reply:
Yes, I will do that later today or else tomorrow.

Reply:
@nicolas-grekas done. There are no `UPGRADE-4.2.md` yet, so I've created one.

Reply:
Never mind, there are already was an `UPGRADE-4.2.md` on master, I've just rebased against master.